Xamarin.Forms.Core/ResourceDictionary.cs

```diff
@@ -13,7 +14,26 @@ public class ResourceDictionary : IResourceDictionary, IDictionary<string, objec
 		static ConditionalWeakTable<Type, ResourceDictionary> s_instances = new ConditionalWeakTable<Type, ResourceDictionary>();
 		readonly Dictionary<string, object> _innerDictionary = new Dictionary<string, object>();
 
+#pragma warning disable 618
+		[TypeConverter(typeof(TypeTypeConverter))]
+		public Type Source
```

Comment:
if `Source` is the new name, and `MergedWith` the deprecated one, the actual implementation should be on `Source` with `MergedWith` proxying it for backward compatibility. not the other way around.